conf/base.config

```diff
@@ -60,10 +59,22 @@ process {
     }
     // TODO nf-core: Customise requirements for specific processes.
     // See https://www.nextflow.io/docs/latest/config.html#config-process-selectors
-    withName:HIFIASM {
-        cpus   = { check_max( 28    * task.attempt, 'cpus'    ) }
+    withName:'.*HIFIASM' {
```

Comment:
`nextflow run genomeassembly/ -profile docker,test_github`
With above changes, the jobs still failed on my laptop, and the exit code is 137.

Reply:
The entry for HIFIASM_PRI in trace file says 'memory' is 6G but 'peak_rss' is 16.1 GB

`nextflow run -ansi-log true main.nf -profile test,singularity --outdir test`

`/lustre/scratch124/tol/projects/darwin/users/kk16/development/nextflow/dev/genomeassembly/test/genomeassembly_info/execution_trace_2023-11-30_09-44-50.txt`

Reply:
I've noticed that too: sometimes the "peak_rss" is higher than the memory that was requested but the job still completed.
I think it's because LSF monitors jobs by polling, say every minute. If the job is quick enough, it may run without LSF noticing. And indeed, your two jobs took less than 30 seconds.
Maybe I was just unlucky when I ran it yesterday :)